DuggaSys/microservices/shared_microservices/getUid_ms.php

```diff
@@ -0,0 +1,28 @@
+<?php
+    date_default_timezone_set("Europe/Stockholm");
+    
+    // Include basic application services
+    include_once ("../../../../coursesyspw.php");
```

Comment:
Not sure if this is used in this file? If removed an SQL error occurs, but it might be because it is needed in `editBoxTitle_ms.php` and therefore maybe should be included there instead?

Reply:
It is used, when removed it causes error 500